templates/typo3/.platform.template.yaml

```diff
@@ -9,6 +9,7 @@ info:
   tags:
   - PHP
   - CMS
+  - Typo3
```

Comment:
As for Sylius, Typo3 is using Symfony (components only) so i don't know if we need to make SF appears here... 
if yes, it should appears also for Drupal templates, right ?

Reply:
Ah I wasn't aware of this. Yes, Symfony might appear here as well, but I'm not sure this makes sense (same for Drupal)

Reply:
Sylius is Symfony full stack, Drupal and Typo3 are not

Reply:
true, so lets forget about all those comments...